### PR TITLE
Fix Cycle bug and update selected cycle in firebase

### PR DESCRIPTION
**Pull request by jpoist97**
Fixed bug where cycles would not show up after reloading app. Also added functionality so that you can not select a cycle and it's updated in firebase.

### CODE REVIEW

File: components/Home/Home.jsx [hunk 1]
```diff
@@ -128,37 +128,36 @@ export default () => {
...
       }
     });
-    return workoutList.slice(0,5);
-  }
-
+    return workoutList.slice(0, 5);
+  };
 
   const selectedCycle = (cycles.selectedCycleId !== undefined) && _.find(cycles.cycles, (cycle) => cycle.id === cycles.selectedCycleId);
   let cycleDetails;
```
Comment (ctperry0301) [inline components/Home/Home.jsx]: You will get an index undefined error if you try to click on a left or right arrow on the home page if a cycle isn't set

Comment (ctperry0301) [reply components/Home/Home.jsx]: > You will get an index undefined error if you try to click on a left or right arrow on the home page if a cycle isn't set

Im also getting the same error if I try to create a cycle while not having a cycle currently selected (new account).

**Review by ctperry0301 — CHANGES_REQUESTED** [review]
I dont think this code is working as expected for me, The cycles arent getting loaded in for me when I try and test it.

**Comment by jpoist97** [comment]
Thanks for catching that Cole, I forgot to test making a new account. All the bugs should be fixed now

**Review by ctperry0301 — APPROVED** [review]
Good Stuff!

**Pull request merged**